src/frontend/src/store/modules/notices.js

```diff
@@ -86,6 +87,31 @@ export default {
         console.log(error);
       }
     },
+    async FETCH_LANGUAGES({ commit }) {
+      try {
+        const { data } = await getAction(`/api/notices/languages`);
+        commit("SET_LANGUAGES", data.languages);
+      } catch (error) {
+        console.log(error);
+      }
+    },
```

Comment:
```suggestion
    async FETCH_LANGUAGES({ commit }) {
        const { data } = await getAction(`/api/notices/languages`);
        commit("SET_LANGUAGES", data.languages);
    },
```

코일이 준 코치한테 물어봤을때 actions 에서는 예외처리를 하는 의미가 없다고 하네요.
api에서 1차로 통신에 대한 try-catch를 잡고, component에서 dispatch할 때 try-catch를 한번 더 잡으면 actions에서는 예외를 안잡아도 될것 같은데 앨런은 어떻게 생각하시나요?

Reply:
저도 다른코드를 보고 따라쳤었는데요..! 예외를 2번 캐치하고 있었군요
현재 notice 관련 actions의 try catch를 모두 제거할게요~